src/components/LoginPage/LoginPageComponent.js

```diff
@@ -1,67 +1,159 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import slide1 from 'components/LoginPage/images/slide-1.png';
-import slide2 from 'components/LoginPage/images/slide-2.png';
-import slide3 from 'components/LoginPage/images/slide-3.png';
-import slide4 from 'components/LoginPage/images/slide-4.png';
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { respondTo } from "_respondTo";
+import slide1 from "components/LoginPage/images/slide-1.svg";
+import slide2 from "components/LoginPage/images/slide-2.svg";
+import slide3description from "components/LoginPage/images/slide-3-description.svg";
+import slide4 from "components/LoginPage/images/slide-4.png";
+
+console.log("slide1", slide1);
+
+const OuterWrapper = styled.div`
+  background-color: #131c3e;
+`;
+
+const MainHeader = styled.h1`
+  display: block;
+  font-family: Roboto;
+  font-weight: 100;
+  font-size: 1.5rem;
+  line-height: 42px;
+  text-align: center;
+  color: #ffffff;
+  padding-top: 40px;
+  padding-bottom: 50px;
+
+  b {
+    color: #96ffff;
+  }
+
+  ${respondTo.xs`
+    font-size: 2rem;
```

Comment:
It sounds like we want to have `2rem` font size for extra small devices.

But in fact we'll have `2rem` font size for small devices and up.

Can we make the code clearer?